test/test_write_concern.py

```diff
@@ -0,0 +1,50 @@
+import collections
```

Comment:
We need to preface this code with the copyright/license note. You can use the following text:

```
# Copyright 2018-present MongoDB, Inc.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
Done.

I suggest that you add a presubmit check (or a test on Travis) making sure that all files are prefaced with a copyright note, that will save future contributors some time and save you some peace of mind.

Reply:
I'll look into creating a presubmit check, thanks for your suggestion.